ethereum/core/src/test/java/org/hyperledger/besu/ethereum/vm/VMReferenceTest.java

```diff
@@ -65,53 +68,24 @@
   // fully test these operations and the mocking does not add much value.
   // Additionally, the GeneralStateTests provide coverage of these
   // operations so the proper functionality does get tested somewhere.
-  private static final String[] BLACKLISTED_TESTS = {
-    "balance0",
-    "balanceAddressInputTooBig",
-    "balanceCaller3",
-    "balanceAddressInputTooBigRightMyAddress",
-    "ExtCodeSizeAddressInputTooBigRightMyAddress",
-    "env1",
-    "extcodecopy0AddressTooBigRight",
-    "PostToNameRegistrator0",
-    "CallToReturn1",
-    "CallRecursiveBomb0",
-    "createNameRegistratorValueTooHigh",
-    "suicideNotExistingAccount",
-    "callstatelessToReturn1",
-    "CallRecursiveBomb1",
-    "ABAcallsSuicide1",
-    "suicideSendEtherToMe",
-    "suicide0",
-    "CallToNameRegistrator0",
-    "callstatelessToNameRegistrator0",
-    "PostToReturn1",
-    "callcodeToReturn1",
-    "ABAcalls0",
-    "CallRecursiveBomb2",
-    "CallRecursiveBomb3",
-    "ABAcallsSuicide0",
-    "callcodeToNameRegistrator0",
-    "CallToPrecompiledContract",
-    "createNameRegistrator"
+  private static final String[] IGNORED_TESTS = {
+    "push32AndSuicide", "suicide", "suicide0", "suicideNotExistingAccount", "suicideSendEtherToMe",
```

Comment:
Why did we have so many more tests ignored before and why are these still ignored?

Reply:
Self descruct tests are known to be problematic for all clients.